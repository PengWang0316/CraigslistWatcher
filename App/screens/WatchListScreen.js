import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

import DarkGreenStatusBar from '../components/DarkGreenStatusBar';

const styles = StyleSheet.create({
  mainView: { backgroundColor: '#e0e0e0', flex: 1 }
});

const WatchListScreen = props => (
  <View style={styles.mainView}>
    <DarkGreenStatusBar />
    <Text>The WatchListScreen</Text>
  </View>
);
export default WatchListScreen;
