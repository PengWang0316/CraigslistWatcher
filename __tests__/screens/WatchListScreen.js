import React from 'react';
import renderer from 'react-test-renderer';

import WatchListScreen from '../../App/screens/WatchListScreen';

jest.mock('../../App/components/DarkGreenStatusBar', () => 'DarkGreenStatusBar');

describe('Test WatchListScreen', () => {
  test('snapshot test', () => expect(renderer.create(<WatchListScreen />).toJSON()).toMatchSnapshot());
});
