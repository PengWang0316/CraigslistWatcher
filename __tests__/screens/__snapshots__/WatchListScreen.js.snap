// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test WatchListScreen snapshot test 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e0e0e0",
      "flex": 1,
    }
  }
>
  <DarkGreenStatusBar />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    The WatchListScreen
  </Text>
</View>
`;
